zaas-service/src/test/java/org/zowe/apiml/zaas/zaas/ExtractAuthSourceFilterTest.java

```diff
@@ -10,23 +10,21 @@
 
 package org.zowe.apiml.zaas.zaas;
 
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 import org.springframework.security.authentication.InsufficientAuthenticationException;
+import org.zowe.apiml.security.common.error.AuthExceptionHandler;
+import org.zowe.apiml.security.common.token.TokenNotValidException;
 import org.zowe.apiml.zaas.security.service.schema.source.AuthSource;
 import org.zowe.apiml.zaas.security.service.schema.source.AuthSourceService;
 import org.zowe.apiml.zaas.security.service.schema.source.JwtAuthSource;
 import org.zowe.apiml.zaas.security.service.schema.source.ParsedTokenAuthSource;
-import org.zowe.apiml.security.common.error.AuthExceptionHandler;
-import org.zowe.apiml.security.common.token.TokenNotValidException;
-
-import jakarta.servlet.FilterChain;
-import jakarta.servlet.ServletException;
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
-import org.zowe.apiml.zaas.zaas.ExtractAuthSourceFilter;
 
```

Comment:
unused

Reply:
Sorry, wrong comment here. I don't know if the other got published but I was commenting about having this gateway loader path in SCGW